subprojects/core/src/main/java/org/gradle/initialization/DefaultParallelConfiguration.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.initialization;
```

Comment:
If I remember correctly, I use the following logic. Daemon properties are contained inside the launcher project as the client is located there. The parallel logic is inside the core, so should the property parsing. I'm open to change this if you feel it make more sense.